windows-driver-docs-pr/devtest/28720-banned-api-usage-use-updated-function-replacement.md

```diff
@@ -12,7 +12,7 @@ f1_keywords:
 **Warning C28720: IsBadXXXPtr API Usage**\
 Example output: ```Banned API Usage:  *function name* is a Banned API as listed in dontuse.h for security purposes.```
 
-Using one of these APIs indicates that you are testing to see if memory is mapped, which implies that you either don't own the memory at all (so you shouldn't be trying to access it), or it indicates a need for better [object lifetime management](https://docs.microsoft.com/en-us/cpp/cpp/object-lifetime-and-resource-management-modern-cpp?view=msvc-170) (if you need to test if something still exists). 
+Using one of these APIs indicates that you are testing to see if memory is mapped, which implies that you either don't own the memory at all (so you shouldn't be trying to access it), or it indicates a need for better object lifetime management (if you need to test if something still exists). For more information on the latter, see [Object Lifetime and Resource Management (RAII)](https://docs.microsoft.com/en-us/cpp/cpp/object-lifetime-and-resource-management-modern-cpp?view=msvc-170). 
```

Comment:
I think the URL here should be a relative path since it's a Microsoft doc: `/cpp/cpp/object-lifetime-and-resource-management-modern-cpp`

Reply:
Done!